scripts/examples/gn_efr32_example.sh

```diff
@@ -180,6 +182,10 @@ else
                 optArgs+="use_silabs_thread_lib=true chip_openthread_target=$SILABS_THREAD_TARGET openthread_external_platform=\"""\" use_thread_coap_lib=true "
                 shift
                 ;;
+            --use_lwip_lib)
+                optArgs+="lwip_root=\""//third_party/connectedhomeip/third_party/silabs"\" "
```

Comment:
I think it would be preferable to use the gsdk lwip default and have the option to use matter version if need be. 
@jmartinez-silabs what do you think?

Reply:
That would be best indeed

Reply:
I would agree too

Reply:
Changed the flag name to `--use_chip_lwip_lib` to reflect the default lwip